Misc/NEWS.d/next/Core and Builtins/2017-07-23-11-28-45.bpo-26579.lpCY8R.rst

```diff
@@ -0,0 +1,7 @@
+Added ``object.__getstate__`` which provides the default implementation of
+the ``__getstate__()`` method.
+
+Copying and pickling instances of subclasses of builtin types bytearray,
```

Comment:
Interpreted text roles could be used here.

Reply:
Yes, but I afraid that it would add too much noise. All links are added in What's New.